src/analysis/problematic_exclusions.py

```diff
@@ -72,9 +72,11 @@ def load_and_format_tsv(path: str, prefix_map: Dict[str, str]) -> pd.DataFrame:
     return df2
 
 
+# TODO: refactor to use utils.py get_labels()
 # todo: In the future, refactor to use OAK.
+#
 # noinspection DuplicatedCode
-def get_labels(
+def populate_labels(
```

Comment:
## Questions: Missing labels
### 1. `mirror_signature` deprecated
Is it expected that the `mirror_signature` files contain deprecated terms? Right now deprecated terms can appear there. That is what I am using as the starting point for collecting terms. The logic I'm using for determining mappable terms that are still unmapped is: _(terms in the `mirror_signature`) - (excluded terms) - (already mapped terms)_.

Reply:
fixed; w/ nico's help i made a query for deprecated